### PR TITLE
Ac 32436 terraform upgrade version

### PR DESCRIPTION
**Pull request by yuvalstyrbrod**
-  new terraform version 
-  remove vendor
-  fix arm link

### CODE REVIEW

File: provider/resource_access_policy.go [hunk 1]
```diff
@@ -105,6 +114,66 @@ func LuminateAccessPolicyBaseSchema() map[string]*schema.Schema {
...
+						Type:        schema.TypeList,
+						Optional:    true,
+						Description: "list of managed devices that have restriction access",
+						Elem: &schema.Resource{
+							Schema: map[string]*schema.Schema{
+								"opswat": {
+									Type:         schema.TypeBool,
+									Optional:     true,
+									Default:      false,
+									Description:  "Indicate whatever to restrict access to Opswat MetaAccess",
```
Comment (gitomics) [inline provider/resource_access_policy.go:127]: whatever -> whether

Comment (gitomics) [reply provider/resource_access_policy.go:127]: Indicate whatever to restrict access to Opswat MetaAccess

Indicate whether to restrict access according to Opswat MetaAccess

File: provider/resource_tcp_application.go [hunk 2]
```diff
@@ -197,7 +201,6 @@ func setTCPApplicationFields(d *schema.ResourceData, application *dto.Applicatio
 	d.Set("type", application.Type)
 	d.Set("visible", application.Visible)
 	d.Set("notification_enabled", application.NotificationsEnabled)
-	d.Set("internal_address", TCPApplicationInternalAddress)
```
Comment (gitomics) [inline provider/resource_tcp_application.go:200]: safe to remove?

File: provider/resource_web_access_policy.go [hunk 3]
```diff
@@ -13,72 +14,32 @@ import (
 func LuminateWebAccessPolicy() *schema.Resource {
 	webSchema := LuminateAccessPolicyBaseSchema()
 
-	conditionsResource := webSchema["conditions"].Elem.(*schema.Resource)
```
Comment (gitomics) [inline provider/resource_web_access_policy.go:16]: safe to remove?

File: service/dto/access_policy_converter.go [hunk 4]
```diff
@@ -186,31 +186,19 @@ func ConvertToDto(accessPolicy *AccessPolicy) sdk.PolicyAccess {
 			})
 		}
 
-		if accessPolicy.Conditions.ManagedDevice.SymantecCloudSoc || accessPolicy.Conditions.ManagedDevice.SymantecWebSecurityService {
```
Comment (gitomics) [inline service/dto/access_policy_converter.go:189]: safe to remove?

**Review by gitomics — CHANGES_REQUESTED** [review]

File: provider/resource_web_access_policy.go (hunk 3 above)
Comment (yuvalstyrbrod) [reply provider/resource_web_access_policy.go:16]: we have conditions as part of the baseAccessPolicyBaseSchema

File: service/dto/access_policy_converter.go (hunk 4 above)
Comment (yuvalstyrbrod) [reply service/dto/access_policy_converter.go:189]: i'm not removing it getting it from a function getDeviceArguments()

File: provider/resource_tcp_application.go (hunk 2 above)
Comment (yuvalstyrbrod) [reply provider/resource_tcp_application.go:200]: we are putting a dummy value "tcp://luminate-dummy:22" ,
don't want it to be part of the user state

**Review by gitomics — APPROVED** [review]

**Pull request merged**